### ISSUE TITLE
Adding planet's temperature

### ISSUE THREAD

**Issue by hamzahsarwari** [Zac-Garby/solar-system#9]
Is there any way you can make it possible to have populations of the planets as they move around. However, they are constant temperatures set into the code.

**Comment by zac-garby** [comment]
Yeah, that could work. Do you mean a different temperature depending on their position in the orbit, or a constant random temperature for each planet?
